tests/integration/test_studio.py

```diff
@@ -127,7 +124,12 @@ def test_custom_image(self):
         self.assertFalse(radio_buttons['auto'].is_selected())
         url_field = self.background_image_url_field
         self.assertEqual(url_field.get_attribute('value'), '')
-        self.assertEqual(self.target_preview_img.get_attribute('src'), default_bg_img_src)
+        self.assertIn(
+            default_bg_img_src.split('http://localhost:8081/')[1], self.target_preview_img.get_attribute('src')
+        )
+
+        custom_bg_img_src = '{}?my-custom-image=true'.format(self.target_preview_img.get_attribute('src'))
```

Comment:
Due to port issue link was broken here.

Reply:
@feanil I am not sure the reason behind the change of port. The above piece of code will fix the broken test as we are not comparing ports anymore. But if you want us to find out the root cause of this failure, please create a YONK ticket. 
cc: @awais786

Reply:
@ihtram thanks for review. I will wait for @feanil's answer before taking further action.

Reply:
@ihtram I think the issue is probably with how the test system is started up and it no longer having a default port.  I think this change is good and we don't need to dig much deeper.  I'm not sure the value of fixing it is worth the time.